packages/trader/src/Modules/Reports/Constants/data-table-constants.js

```diff
@@ -111,7 +113,7 @@ export const getProfitTableColumnsTemplate = (currency, items_count) => [
         renderCellContent: ({ cell_value, is_footer }) => {
             if (is_footer) return '';
 
-            return <Money amount={cell_value} currency={currency} />;
+            return <Money amount={cell_value} currency={currency} should_format_crypto={0} />;
```

Comment:
can we pass boolean instead?

Reply:
As we need to send a falsy value here and the convention for passing booleans to components is just ignoring them, I decided to use a number instead.

Reply:
I guess we can change it [here](https://github.com/binary-com/deriv-app/pull/1854/files#diff-9bf871399a9c3968ee9be52517a27ddfR21) to accept boolean for `should_format_crypto`. It's better to use boolean IMO since this can only be `true` or `false`.